swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+
+    def add(self, added_item):
+        self.inventory.append(added_item)
+        return added_item
+
+
+    def remove(self, removed_item):
+        if removed_item not in self.inventory:
+            return False 
+        else:
+            self.inventory.remove(removed_item)
+            return removed_item  
+
+
+    def get_by_category(self, category=""):
+        items = []
+        for item in self.inventory:
+            if category == item.category:
+                items.append(item)
+        return items
+    
+
+    def swap_items(self, vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in vendor.inventory:
+            return False
+        else:
+            self.remove(my_item)
+            vendor.add(my_item)
+            vendor.remove(their_item)
+            self.add(their_item)
+            return True
```

Comment:
same as above! Let's remove the `else` and let the main focus of the method stand on its own